home.css

```diff
@@ -0,0 +1,449 @@
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.doDivImg {
+  margin-left: 10px;
+}
+
+.doDivHead {
+  font-size: 25px;
+}
+
+.doheadHr {
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+  margin-bottom: 20px;
+}
+
+.doivpara {
+  color: white;
+  margin-left: 10px;
+}
+
+.doInno {
+  background-color: red;
+  margin-left: 30px;
+  margin-right: 30px;
+  height: 80px;
+  text-align: center;
+  display: block;
+}
+
+.doLink {
+  display: none;
+}
+
+.doInnoPara {
+  margin-top: 40px;
+  padding-top: 30px;
+  font-size: 25px;
+}
+
+.ourTeam {
+  text-align: center;
+  color: black;
+  font-size: 20px;
+}
+
+.ourHr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+}
+
+.teamGrid {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 10px;
+}
+
+.teamDec {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid red;
+}
+
+.avatar1 {
+  height: 100px;
+  width: 100px;
+  margin-left: 10px;
+}
+
+.teamDet {
+  list-style-type: none;
+}
+
+.teamHr {
+  margin-right: 190px;
+}
+
+.titleT {
+  color: red;
+}
+
+footer {
+  margin-top: 100px;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(58, 56, 56);
+  padding: 10px;
+}
+
+.partnersDiv {
+  text-align: center;
+  padding: 20px 0;
+  font-size: 30px;
+}
+
+.partOn {
+  display: flex;
+  justify-content: center;
+  gap: 80px;
+}
+
+.partTw {
+  justify-content: center;
+  gap: 80px;
+  display: flex;
+}
+
+.div1 {
+  color: white;
+}
+
+.div1Hr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.div2 {
+  color: rgb(190, 195, 199);
+}
+
+.div4 {
+  color: rgb(142, 145, 148);
+}
+
+.div5 {
+  color: rgb(190, 195, 199);
+}
+
+.div3 {
+  color: rgb(101, 103, 104);
+}
+
+.copyDiv {
+  display: flex;
+}
+
+.copyright {
+  display: flex;
+  margin-top: 90px;
+  justify-content: center;
+  align-items: center;
+  margin-left: 20px;
+  gap: 30px;
+}
+
+.plugicon {
+  height: 50px;
+}
+
+@media only screen and (min-width: 768px) {
+  .sideMback {
+    display: none;
+  }
+
+  .navSlide {
+    display: block;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .mediNbac {
+    background-color: black;
+    height: 30px;
+  }
+
+  .mediaNav {
+    display: flex;
+    float: right;
+    margin-right: 190px;
+    margin-top: 3px;
+  }
+
+  .mediaNav > li {
+    display: inline;
+    color: white;
+    margin-left: 10px;
+  }
+
+  .naVimg {
+    height: 18px;
+  }
+
+  .relNav {
+    display: flex;
+    background-color: white;
+    justify-content: center;
+    gap: 160px;
+    height: 60px;
+  }
+
+  .Menuli:hover {
+    border: 2px solid red;
+    color: red;
+  }
+
+  .Navle {
+    display: flex;
+    list-style-type: none;
+    margin-top: 0;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .NavRig {
+    float: left;
+    display: flex;
+    list-style-type: none;
+    gap: 40px;
+    margin-top: 0;
```

Comment:
- [ ] There are still some class names that start with uppercase letters please, make them lower case as the previous reviewer suggested.